site/styles.css

```
@@ -200,1010 +234,781 @@ code {
+  background: color-mix(in srgb, var(--ink-900) 55%, transparent);
 }
 
-.wg-footer__top,
-.wg-footer__bottom {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  flex-wrap: wrap;
-  gap: var(--space-lg);
+@media (max-width: 620px) {
 }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)